projects/Amlogic/options

```diff
@@ -34,24 +34,35 @@
   # Displayserver to use (x11 / no)
     DISPLAYSERVER="no"
 
-  # Windowmanager to use (ratpoison / fluxbox / none)
+  # Windowmanager to use (fluxbox / none)
     WINDOWMANAGER="none"
 
   # additional Firmware to use (dvb-firmware, misc-firmware, wlan-firmware)
-    FIRMWARE="brcmfmac_sdio-firmware meson-firmware"
+  # Space separated list is supported,
+  # e.g. FIRMWARE="dvb-firmware misc-firmware wlan-firmware"
+    FIRMWARE="isc-firmware wlan-firmware dvb-firmware brcmfmac_sdio-firmware meson-firmware"
```

Comment:
The "misc"-firmware and wlan-firmware packages are not needed on these devices (the latter might conflict with brcmfmac_sdio-firmware) and dvb-firmware package is only needed if the matching modules are configured in the kernel defconfig? (not something I have knowledge to check). There are some outstanding PRs to review on dvb-firmware btw.

Reply:
- removed the `isc-firmware wlan-firmware` packages
- `dvb-firmware` is still there because the necessary modules are already build at the image just no firmware files are available at the image, makes not much sense to not include them